README.md

```diff
@@ -236,6 +236,25 @@ case it's a new package, be sure to add it to the correct koji tags as well.  [H
 
 * This uses local filesystem, so please be sure your using a clean checkout with no local changes
 
+## Releasing Katello client packages for SUSE
+
+For SLES client builds, they must be manually done for now by submitting the jobs to [OBS](https://build.opensuse.org/).  You can use the OSC client tools.  All Katello packages are hosted in the [systemsmanagement:katello](https://build.opensuse.org/project/show/systemsmanagement:katello) project.
+
+After installing OSC, you can check out the particular package you're interested in.  The first time OSC runs, it will prompt for and save your credentials.
+
+
+```
+osc checkout systemsmanagement:katello katello-repos
+```
+
+Update the files there in, as needed. For SLES packages, you need to ensure the `Release` line is in the following format.  Otherwise the spec files will build on both Koji and OBS.
+
+```
+Release:        <CI_CNT>.<B_CNT>.%{?dist}
+```
+
+Add and commit the changes.  OBS packages will be built automatically, and once complete, you can download them and import into koji.
+
```

Comment:
Mind documenting how to import them into koji and then tagging them?

Reply:
Updated